src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -175,27 +203,48 @@ full name
                 IsGlobal = true,
                 Name = TombstonesSlice
             });
-
             TombstonesSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef
             {
                 StartIndex = 1,
                 IsGlobal = false,
                 Name = CollectionEtagsSlice
             });
-
             TombstonesSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef
             {
                 StartIndex = 1,
                 IsGlobal = true,
                 Name = AllTombstonesEtagsSlice
             });
-
             TombstonesSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef()
             {
                 StartIndex = 2,
                 IsGlobal = false,
                 Name = DeletedEtagsSlice
             });
+
+
+            // The attachments schema is as follows
+            // 7 fields (lowered document id, recored separator, lowered name, etag, name, content type, last modified, stream identifier)
+            // We are you using the record separator in order to avoid loading another files that has the same key prefix, 
+            //      e.g. fitz(record-separator)profile.png and fitz0(record-separator)profile.png, without the record separator we would have to load also fitz0 and filter it.
```

Comment:
I'm not following. With a separator I can see by prefix: fitz(record-separator) and get all the attachments of it without attachments of fitz0.